code/alden/python/lab09.py

```diff
@@ -0,0 +1,67 @@
+import math
+class ATM():                
+    def __init__(self):             # In initializer put balance, interest rate, and placeholder for transactions.
+        self.__balance = 0
+        self.interest = 0.001
+        self.transactions = []
+        
+    def check_balance(self):        # Returns current balance when called.
+        self.transactions.append(f'Check balance: ${self.__balance}')
```

Comment:
* Great addition of adding "Check Balance" as a transaction. It super easy to log stuff after we have an object (self.transactions). Just append the new string to the list.